pubspec.yaml

```diff
@@ -8,5 +8,5 @@ environment:
 dependencies:
   enumerators: '>=0.6.0 <0.7.0'
 dev_dependencies:
-  collection: '>=0.9.3 <0.10.0'
+  collection: '>=0.9.3 <1.15.0'
```

Comment:
Shouldn't we also bump the lower bound? I'm unable to browse the API docs for versions <1.4.0 but I suppose that in version 0.9.3 `equality.dart` wasn't re-exported by `collection.dart`. At least, there's a chance it wasn't because there's a deprecation warning saying that now users should import `collection.dart` instead. I think it's fine to bump it to 1.4.0, I don't think anyone is using these old versions anyway.

Reply:
I can update the lower bound as well. I was just trying to allow the package to work with the original dependencies as well as the latest, if possible.

I'll update the lower bound now.